### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ZAAFHachemrachid** [prasanthrangan/hyprdots#1044] [Q&A]
hello everyone ,  in login screen i want change and add images in that folder 
![IMG20240309133806](https://github.com/prasanthrangan/hyprdots/assets/117745305/4ff46dbe-50a7-47dd-9290-d0a85a01bf53)
![IMG20240309133753](https://github.com/prasanthrangan/hyprdots/assets/117745305/a5bceda5-bf46-49b0-b1d3-f0f4f38a2bd0)
![IMG20240309133806](https://github.com/prasanthrangan/hyprdots/assets/117745305/ca0a702a-a154-49ab-be72-aea60cd7e1c6)

**Comment by kRHYME7** [comment]
Corners 

``` /usr/share/sddm/themes/Corners/backgrounds ```
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6a6af730-4b77-47e8-ac14-22aac8cd2fa9)





If theme is ``` candy ```
``` /usr/share/sddm/themes/Candy/backgrounds/ ```
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9027d30d-a704-4448-8abc-272a51e7f045)

**Comment by ZAAFHachemrachid** [comment]
THX BRO
